src/pgduckdb_utils.cpp

```diff
@@ -1,19 +1,13 @@
 #include "pgduckdb/pgduckdb_utils.hpp"
 
-extern "C" {
-#include "postgres.h"
-#include "miscadmin.h"
-#include "lib/stringinfo.h"
-#include "storage/fd.h"
-#include "executor/spi.h"
-}
-
 #include <string>
 
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
-#include <errno.h>
+
+// FIXME - this file should not depend on PG, rather DataDir should be provided
+extern PGDLLIMPORT char *DataDir;
```

Comment:
Is this to avoid including any postgres headers here?

Reply:
I'm surprised `PGDLLIMPORT` is available without importing postgres headers. I guess that's because `pgduckdb_utils.hpp` is still importing `postgres.h`.